### ISSUE TITLE
BUG: Deleted callout visible in Top Callouts

### ISSUE THREAD

**Issue by larssondenise** [alkem-io/client-web#4220]
**Describe the bug**
When I delete the Callout, they remain visible in the top Callouts box. Also after refreshing the page. For users who are trying and testing the platform, they expect the callouts to be gone from this list after deletion and it will block them from playing with the platform.

**To Reproduce**

1. Go to https://alkem.io/uwv/challenges/seriousgame-cultu-5688/dashboard
2. Callout is visible in Top Callouts but not under Contribute

**Expected behavior**
Deleted Callouts not visible in top callouts

**Comment by techsmyth** [comment]
@Comoque1 can you please investigate this one?

**Comment by Comoque1** [comment]
@techsmyth, @larssondenise,  Tested on ACC env and cannot reproduce the issue for Space / Challenge / Opportunity. The p callouts block is removed as well, after removing all callouts:

![image](https://github.com/alkem-io/client-web/assets/23620658/f65ef7c6-907f-4867-a4b9-42ed9f92051b)
![image](https://github.com/alkem-io/client-web/assets/23620658/6fc73342-c377-492f-b8b6-565d617bd55e)
![image](https://github.com/alkem-io/client-web/assets/23620658/ba25df70-2696-422f-804f-9c4c5465e79e)


I would suggest to close the issue and if reproducible, after we update production - reopen it.

**Comment by techsmyth** [comment]
yes agreed that this can be closed. My suspicion is that this was a dataloader issue, so if not easily reproducible then it would at least go away after the cached version on the server expires.
